mobile/src/libs/maps/icon-name-to-icon.map.ts

```diff
@@ -0,0 +1,20 @@
+import BackArrowIcon from '#assets/img/icons/back-arrow.svg';
+import BellIcon from '#assets/img/icons/bell.svg';
+import ChatIcon from '#assets/img/icons/chat.svg';
+import HomeIcon from '#assets/img/icons/home.svg';
+import PlusIcon from '#assets/img/icons/plus.svg';
+import UserIcon from '#assets/img/icons/user.svg';
+import { type IconName } from '#libs/types/icon-name.type';
+
+type SVGIcon = React.FC<React.SVGProps<SVGSVGElement>>;
+
+const iconNameToIcon: Record<IconName, SVGIcon> = {
```

Comment:
it would be better to refer into icon package

Reply:
@v0ldemar01 please could you explain what do you mean ?) we need to have the same structure with frontend, thats why i created it like this and content of this file working good for now, as before i had different errors

Reply:
If we have an icon component in libs/components/icon so this map only linked to that components, so we don't need to expose as common map